pyatlan/pkg/widgets.py

```diff
@@ -0,0 +1,1037 @@
+        hidden: StrictBool = False,
+        help: StrictStr = "",
+        grid: StrictInt = 8,
+    ):
+        """
+        Widget that allows you to generate a unique key that could be used for securing an exchange or other unique
+        identification purposes, and provides buttons to regenerate the key or copy its text. Will return the generated
+        key as clear text.
+
+        :param label: name to show in the UI for the widge
+        :param required: whether a value must be selected to proceed with the UI setup
+        :param hidden: whether the widget will be shown in the UI (false) or not (true)
+        :param help: informational text to place in a hover-over to describe the use of the input
+        :param grid: sizing of the input on the UI (8 is full-width, 4 is half-width)
+        """
+        widget = KeygenInputWidget(
+            label=label,
+            hidden=hidden,
+            help=help,
+            grid=grid,
+        )
+        super().__init__(type_="string", required=required, ui=widget)
+
+
+@dataclasses.dataclass
+class MultipleGroupsWidget(AbstractWidget):
+    def __init__(self, label: str, hidden: bool = False, help: str = "", grid: int = 8):
+        super().__init__(
+            widget="groupMultiple",
+            label=label,
+            hidden=hidden,
+            help=help,
+            grid=grid,
+        )
+
+    def get_validator(self, name: str):
+        return (
+            f'{name}:Optional[list[str]] = Field(default_factory=list)\n")'
+            f"_validate_{name} = validator("
+            "{name}"
+            ', pre=True, allow_reuse=True)(validate_multiselect)\n")'
+        )
+
+
+@dataclass
+class MultipleGroups(AbstractUIElement):
+    @validate_arguments()
+    def __init__(
+        self,
+        label: StrictStr,
+        required: StrictBool = False,
+        hidden: StrictBool = False,
+        help: StrictStr = "",
+        grid: StrictInt = 8,
+    ):
+        """
+        Widget that allows you to choose multiple groups, and returns an array of group names that were selected.
+
+        :param label: name to show in the UI for the widget
+        :param required: whether a value must be selected to proceed with the UI setup
+        :param hidden: whether the widget will be shown in the UI (false) or not (true)
+        :param help: informational text to place in a hover-over to describe the use of the input
+        :param grid: sizing of the input on the UI (8 is full-width, 4 is half-width)
+        """
+        widget = MultipleGroupsWidget(
+            label=label,
+            hidden=hidden,
+            help=help,
+            grid=grid,
+        )
+        super().__init__(type_="string", required=required, ui=widget)
+
+
+@dataclasses.dataclass
+class MultipleUsersWidget(AbstractWidget):
+    def __init__(self, label: str, hidden: bool = False, help: str = "", grid: int = 8):
+        super().__init__(
+            widget="groupMultiple",
```

Comment:
Recently fixed the same typo bug on Java SDK — this one should be `userMultiple`.

Reply:
done